src/main/resources/db/migration/V1_1__grant_to_cloudsqliamuser.sql

```diff
@@ -0,0 +1,7 @@
+DO $$
+BEGIN
+  CREATE ROLE cloudsqliamuser WITH NOLOGIN;
+  EXCEPTION WHEN DUPLICATE_OBJECT THEN
+  RAISE NOTICE 'not creating role cloudsqliamuser -- it already exists';
+END
+$$;
```

Comment:
Og samme for CREATE USER "isyfo-analyse". (se isdialogmote)

Reply:
Lagt til en migration for dette nå.